CMakeLists.txt

```diff
@@ -109,6 +113,8 @@ endif()
 
 if (LINUX)
     include(GNUInstallDirs)
+    set(CMAKE_INSTALL_BACKEND_DIR "backend")
+    set(CMAKE_INSTALL_TEST_DIR "test")
```

Comment:
Do we really want to include test in our release? That changes the requirements for test.

Reply:
Yes.  so that end users can validate the release process